lib/rules/no-old-shims.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const oldShimsData = require('ember-rfc176-data/old-shims.json');
+
+//------------------------------------------------------------------------------
+// General rule - Don't use import paths from ember-cli-shims
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {},
+    fixable: 'code',
+  },
+
+  create(context) {
+    const message = 'Don\'t use import paths from ember-cli-shims';
+
+    return {
+      ImportDeclaration(node) {
+        const source = node.source.value;
+        if (source in oldShimsData) {
+          const moduleMappings = oldShimsData[source];
+
+          const fix = function (fixer) {
+            const newImports = {};
+
+            node.specifiers.forEach((specifier) => {
+              let localName;
+              let importedName;
+              if (specifier.type === 'ImportDefaultSpecifier') {
+                localName = specifier.local.name;
```

Comment:
Looks like we can move `localName` outside of this condition and use const instead:
```
const localName = specifier.local.name;
```